### ISSUE TITLE
Search <start,end> functionality not working 

### ISSUE THREAD

**Issue by GregorKerr1996** [Iceloof/GoogleNews#43]
<img width="309" alt="Screenshot 2020-12-10 at 13 24 01" src="https://user-images.githubusercontent.com/16990550/101777870-fb6e4c00-3aea-11eb-9d6e-028a5b38e446.png">

Start date seems to default to 7days ago? seems to work for dates previous to the start of december? any help would be appreciated to get data from 1st to 3rd

note : if search from 1st to 10th december, still only returns 3-10th

**Comment by sakshiseth** [comment]
@HurinHu If you're not working on this issue. Should I take it up?

**Comment by HurinHu** [comment]
I have checked the google results, it seems the date range is not always correct as what we set, but it will return the date after start date, unless the search result is less than one page on the date range. If you search by period, it will return the proper results on the first and second page. If you go to third page, and they don't have more than 30 items, the result will be out of date range. 
Just like the page show below, it is not always correct on the first page, it should be 1 month ago not 2 weeks ago, but google just return with wrong date. We can't make changes on this at the moment.
![Screen Shot 2021-01-01 at 23 01 35](https://user-images.githubusercontent.com/3132148/103436871-6859a980-4c85-11eb-9631-d4e03455f48d.png)
